App.js

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
+import firebase from 'firebase';
```

Comment:
Have you run the app and tested if it's showing any erro.

Reply:
yes I did